lib/Indexer/Tests/Adapter/ReferenceFinder/IndexedReferenceFinderTest.php

```diff
@@ -206,17 +250,35 @@ public function provideMembers(): Generator
                 <?php namespace Bar; $foo = new Foobar(); $foo->b<>ar();
 
                 // File: project/subject1.php
-                <?php namespace Bar; $foo = new Barfoo(); $foo->bar();
+                <?php namespace Bar; $bar = new Barfoo(); $bar->bar();
 
                 // File: project/class1.php
-                <?php namespace Bar; class Foobar extends Barfoo { public function bar() {}}
+                <?php namespace Bar; class Barfoo { public function bar() {}}
```

Comment:
`public static` according to the description?

Reply:
What description?

Reply:
Yeah the test should actually have a static property. I'm not sure why it doesn't have it. I'll add it. :)

Reply:
Oh there is a break in the git diff just noticed, the test case isn't called static. :laughing: